scripts/fix-status.js

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env node
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+/**
+ * Return a new feature object whose first-level properties have been
```

Comment:
Should this doc block be moved to before the function? Although which function, this seems like it's documenting a helper for sorting keys before JSON-serializing, which isn't here?